docs/Processed Data README.md

```diff
@@ -25,12 +25,6 @@ Each processed text as saved in separate file in format `<tweet_id>.tagget.txt`.
 files contains data provided by `tagger-toyger`, including POS tag, either word is old fashioned or not,
 possible disambiguations, index in the analysed sentenced.
 
-## File `embeddings.pkl`
```

Comment:
Czemu to wszystko zostało wywalone? :(

Reply:
Moje przeoczenie, poszedł commit z przywróconą starą wersją